src/sass/structures/_blog-post.scss

```diff
@@ -32,31 +33,45 @@ body.blog-post {
   }
 
   .page-container {
-    blockquote,
-    ul {
-      clear: both;
+    main {
+      padding: 30px 0;
+      font-weight: 400;
+      font-size: 1.2em;
+      display: block;
+    }
+
+    h2 {
+      font-weight: bolder;
```

Comment:
You will need to update https://github.com/Woorank/woorank-theme/blob/master/src/sass/utils/_variables.scss#L12 to keep track of this "boldness".

Reply:
👍  Thanks Didier!